vms/platformvm/txs/unsigned_tx.go

```diff
@@ -25,6 +25,8 @@ type UnsignedTx interface {
 	Outputs() []*avax.TransferableOutput
 
 	// Attempts to verify this transaction without any provided state.
+	// Note that some other syntactic checks may depend on state so they
+	// are not carried out in this method
```

Comment:
By definition, a check that depends on state isn't syntactic... right?
```suggestion
```

Reply:
Done